docs/design/architectural-overview.md

```diff
@@ -1,9 +1,15 @@
 # Kuadrant Architectural Overview [Draft]
 
+<!--- variables for repeated links --->
+[AuthPolicy]: https://docs.kuadrant.io/kuadrant-operator/doc/auth/
+[RateLimitPolicy]: https://docs.kuadrant.io/kuadrant-operator/doc/rate-limiting/
+[TLSPolicy]: https://docs.kuadrant.io/multicluster-gateway-controller/docs/tlspolicy/tls-policy/
```

Comment:
Prob better to update these to multicluster-gateway-controller/docs/reference/dnspolicy/ multicluster-gateway-controller/docs/reference/tlspolicy/

Reply:
Thanks for the suggestion @maleck13 - updated now